dbms/src/Debug/dbgFuncCoprocessor.cpp

```diff
@@ -56,6 +55,8 @@
 
 #include <utility>
 
+#include <TIDB/Codec/Datum.h>
```

Comment:
I think this include line should be put before `#include <utility>`. Use the `format-diff.py` script to format the files you changed.

https://github.com/pingcap/tiflash#contributing